.teamcityTest/Gradle_Check_Tests/CIConfigIntegrationTests.kt

```diff
@@ -71,21 +72,26 @@ class CIConfigIntegrationTests {
                         if (shouldBeSkipped(subProject, testCoverage)) {
                             return@forEach
                         }
-                        if (subProject.unitTests && testCoverage.testType.unitTests) {
+                        if (subProject.hasOnlyUnitTests()) {
+                            return@forEach
+                        } else if(subProject.unitTests && testCoverage.testType.unitTests) {
```

Comment:
Missing space after `if`.

Reply:
This reminds me of a question, do we have lint check for teamcity Kotlin DSL?

Reply:
If fixed the space. We opened an issue to introduce some linting.